bindings.c

```diff
@@ -2905,11 +2910,11 @@ int cg_rmdir(const char *path)
 
 	controller = pick_controller_from_path(fc, path);
 	if (!controller)
-		return -errno;
+		return errno == ENOENT ? -EPERM : -errno;
```

Comment:
So you referred to `cg_rmdir()` here? I adapted that already in the updated PR but somehow it is still shown here. Not sure why. Just look at the patch https://github.com/lxc/lxcfs/pull/136.patch :)